test/app-decorator.js

```diff
@@ -1075,6 +1075,32 @@ test('extended Schema is an object', async t => {
   await app.ready()
 })
 
+test('array of schemas contains a non-string schema', async t => {
+  const app = Fastify()
+
+  app.register(GQL, {
+    schema: [makeExecutableSchema({
+      typeDefs: `
+          type Query {
+            add(x: Int, y: Int): Int
+          }
+        `,
+      resolvers: {
+        Query: {
+          add: async (_, { x, y }) => x + y
+        }
+      }
+    }
+    )]
+  })
+
+  try {
+    await app.ready()
+  } catch (error) {
+    t.equal(error.message, 'Invalid options: when providing an array to the "schema" option, only string schemas are allowed')
+  }
+})
```

Comment:
this is not the right way to test an exception. if the exception is not raised, this test will succeed

Reply:
Makes sense. I had followed the test pattern in use in this project. For example, in this same test file, you'll find this pattern on lines 336, 1006 and 1025. But, on a second thought, the test would in fact succeed if the exception isn't raised. To prevent this, I updated the test, adding a `t.plan(1)` call.